Crux/Typecheck/Env.hs

```diff
@@ -326,8 +326,8 @@ getAllExportedExceptions :: LoadedModule -> [(Name, TypeVar)]
 getAllExportedExceptions LoadedModule{..} = mconcat $ (flip fmap $ exportedDecls $ mDecls lmModule) $ \case
     DDeclare _ _ _ -> []
     DLet _ _ _ _ _ -> []
-    DFun _ _ -> []
-    DData _ _ _ _ -> []
+    DFun {} -> []
```

Comment:
For the record, I don't use {} because it's too easy to miss important matches when adding a parameter to a variant.